react-components/src/components/RevealToolbar/LayersButton/LayersButton.test.tsx

```diff
@@ -66,6 +66,26 @@ describe(LayersButton.name, () => {
     ).toBe(true);
   });
 
+  test('Layers drop down should mount under the same parent as button', async () => {
+    render(
+      <div data-test-id="layers-button">
+        <LayersButton {...defaultProps} />
+      </div>,
+      {
+        wrapper
+      }
+    );
+
+    const button = screen.getByRole('button', { name: 'Filter 3D resource layers' });
+    await act(async () => {
```

Comment:
You can utilize `userEvent` here instead of using `act`

Reply:
Ah! Thanks. Why is that better?

Reply:
userEvent.click() works with async updates without requiring act() and the other reason `userEvent ` will mimic the user behavior like `button.click` doesn't handle focus while the `userEvent` will add the focus  as it should